paper_figures/code/maxsavings_contour.py

```diff
@@ -12,7 +12,7 @@
     max_tariff_savings, 
     get_start_end
 )
-GENERATE_DATA = False
+GENERATE_DATA = True
```

Comment:
How do we want to leave these flags? Maybe set to `False` in case someone wants to just recreate the plots when they download the repo?

Reply:
good catch! I'll make sure these are false.